go.mod

```diff
@@ -36,11 +30,11 @@ require (
 	k8s.io/klog v1.0.0
 	k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a // indirect
 	k8s.io/kubernetes v1.15.12
-	k8s.io/utils v0.0.0-20191114184206-e782cd3c129f // indirect
 	sigs.k8s.io/controller-runtime v0.2.0
 )
 
 replace (
+	github.com/openebs/lib-csi => github.com/shubham14bajpai/lib-csi v0.0.0-20201218111054-ecb5bfada741
```

Comment:
(Just adding the comment here to remember) this need to be removed once the PR in lib-csi merged.